app/views/lead_providers/your_schools/index.html.erb

```diff
@@ -18,10 +18,13 @@
   </div>
 </div>
 
-<%= render SubnavComponent.new do |component| %>
-  <% @cohorts.each do |cohort| %>
-    <%= component.nav_item(path: lead_providers_your_schools_path(selected_cohort_id: cohort.id)) do %>
-      <%= cohort.start_year %> cohort
+<% if @cohorts.length > 1 %>
+  <%= render SubnavComponent.new do |component| %>
+    <% @cohorts.each do |cohort| %>
+      <%# The link to the current cohort doesn't have it in the URL %>
+      <%= component.nav_item(path: lead_providers_your_schools_path(cohort: cohort.start_year === Time.zone.today.year ? nil : cohort)) do %>
```

Comment:
What happens 1st Jan 2022?

Reply:
Also, why do we not include it for the current cohort?

Reply:
it breaks! will fix

I changed it to not include it to fix the bug where no tab would be selected when initially viewing the page:

![image](https://user-images.githubusercontent.com/472830/122040526-d1665100-cdcf-11eb-8482-528609ceb02a.png)

alternative fix would have been to always ensure the cohort is in the URL, but I felt this was a nicer solution